doc/presentations.rst

```diff
@@ -2,6 +2,34 @@
 External Resources, Videos and Talks
 ===========================================
 
+.. _mooc:
+
+The Official scikit-learn MOOC
+==============================
+
+If you are new to scikit-learn, or looking to strengthen your understanding,
+we highly recommend the **scikit-learn MOOC (Massive Open Online Course)**.
+
+The MOOC, created and maintained by some of the scikit-learn core-contributors,
+is **free of charge** and is designed to help learners of all levels master
+machine learning using scikit-learn. It covers topics
+from the fundamental machine learning concepts to more advanced areas like
+predictive modeling pipelines and model evaluation.
+
+The course materials are available at:
+
+`<https://inria.github.io/scikit-learn-mooc/>`_.
+
+This course is also hosted on the `FUN platform
+<https://www.fun-mooc.fr/en/courses/machine-learning-python-scikit-learn/>`_,
+which additionally makes the content interactive without installation, and gives
```

Comment:
```suggestion
which additionally makes the content interactive without the need to install anything, and gives
```

probably need to fix line length as its too long, but maybe something like this to make it clearer?